### PR TITLE
(Now that the threshold fiasco should be settled) Icons for all

### PR DESCRIPTION
**Pull request by Jac21**

### CODE REVIEW

File: server/index.html [hunk 1]
```diff
@@ -37,4 +55,4 @@
   </div>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
   </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (reddraggone9) [inline server/index.html]: Ah, the neverending fight over whether plain text files should have a newline at the end. Stand strong comrade.

**Review by reddraggone9 — APPROVED** [review]

File: server/index.html (hunk 1 above)
Comment (Jac21) [reply server/index.html]: I appreciate the support.

**Comment by PlasmaPower** [comment]
Merge conflicts

**Comment by Jac21** [comment]
Resolved.

**Pull request merged**